fem/bilinearform.hpp

```diff
@@ -151,8 +151,8 @@ class BilinearForm : public Matrix
    /** This method must be called before assembly. */
    void SetAssemblyLevel(AssemblyLevel assembly_level);
 
-   /// Get the assembly level
-   AssemblyLevel GetAssemblyLevel() {return assembly;}
+   /** Returns the assembly level */
+   AssemblyLevel GetAssemblyLevel() const { return assembly; }
```

Comment:
There is a difference between `/// ...` and `/** ... */` doxygen comments -- we probably should add a section about this in `CONTRIBUTING.md`.

In this case, we want the comment to become a "brief" documentation and since it fits on one line we can use `/// ...` comment. Alternatively, (or if the `/// ...` comment does not fit on one line) one can use `/** @brief ... */` comment to create a "brief" documentation. The comment `/** ... */` without `@brief` becomes a "long" documentation (and so does a multiline `/// ...` comment without `@brief`).

The "brief" documentation is shown next to the method when listed as part of all class methods. The "long" documentation (together with the "brief" documentation) is shown in the detailed method description which can be accessed by clicking the link "More ..." that is added by doxygen at the end of the "brief" documentation.

Doxygen has a number of such conventions that are not obvious, so it is always best to run `make doc` and look at the generated output, e.g. by opening `doc/CodeDocumentation.html` and then searching for the relevant class through the search box in the top right corner.

Reply:
Thank you very much for the information @v-dobrev. I fixed the documentation according to your comments.